### DISCUSSION TITLE
edit Dialog Styling

### DISCUSSION THREAD

**Discussion by TheDemonHuB** [eclipse-theia/theia#12575] [Q&A]
is there any method or functionality to make dialog box resizable or can i adjust the default size for it????  

using class AbstractDialog to create dialog.

**Comment by vince-fugnitto** [comment]
@TheDemonHuB thank you for the discussion, there is no functionality as far as I am aware in the framework as dialogs have fixed sizes through their styling. If you want dialogs to be responsive and resizable you'll likely need to implement it yourself, and make sure their styling supports it.

**Reply by TheDemonHuB** [reply]
can i increase the default fixed size???

**Reply by vince-fugnitto** [reply]
@TheDemonHuB you control the rendering of the dialog, if you need to increase the size then you can do so with CSS for example.

**Reply by TheDemonHuB** [reply]
example?

**Reply by vince-fugnitto** [reply]
> example?

There's really nothing tricky or difficult, if you need to increase the size of a dialog simply style it the way you want to (ex: [width](https://developer.mozilla.org/en-US/docs/Web/CSS/width)). 
For example, our `about dialog` has styling specific for itself:
- https://github.com/eclipse-theia/theia/blob/master/packages/core/src/browser/style/about.css
- https://github.com/eclipse-theia/theia/blob/master/packages/core/src/browser/about-dialog.tsx
